apps/experiments/forms.py

```diff
@@ -285,16 +285,27 @@ class Meta:
 
 
 class TranslateMessagesForm(forms.Form):
+    target_language = forms.ChoiceField(
+        choices=[],
+        required=True,
+        label="Select Language",
+        widget=forms.Select(attrs={"class": "select select-bordered w-full", "id": "translation-language"}),
+    )
     provider_model = forms.ChoiceField(
         choices=[],
         required=True,
         label="Select LLM Provider Model",
         widget=forms.Select(attrs={"class": "select select-bordered w-full", "id": "translation-provider-model"}),
     )
 
-    def __init__(self, *args, team=None, **kwargs):
+    def __init__(self, *args, team=None, translatable_languages=None, widget_id_suffix="", **kwargs):
         super().__init__(*args, **kwargs)
         if team:
             self.fields["provider_model"].choices = [
                 ("", "Choose a model for translation")
             ] + get_dropdown_llm_model_choices(team)
+            self.fields["provider_model"].widget.attrs["id"] = f"translation-provider-model{widget_id_suffix}"
```

Comment:
needed for when the "translate remaining messages" button is also shown because before this change 2 provider_ids were passed and one was blank. I opted for this route rather than filtering out the blank one to ensure we are selecting the one we want

Reply:
Thoughts on having a flag param indicating whether or not this form is meant for translating remaining or all messages. It feels like the API would be cleaner and then inside the form you can do form-specific things depending on that value. So I'm thinking something like
```python
def __init__(self, *args, team=None, translatable_languages=None, is_translate_all_form: bool, **kwargs):
```